src/Microsoft.AspNet.Razor/Generator/LiteralAttributeCodeGenerator.cs

```diff
@@ -62,9 +63,9 @@ public override bool Equals(object obj)
         {
             var other = obj as LiteralAttributeCodeGenerator;
             return other != null &&
```

Comment:
unnecessary. more concretely, `Equals()` remains commutative until and unless someone creates a `LiteralAttributeCodeGenerator` subclass. not likely.